vm/src/hint_processor/builtin_hint_processor/bootloader_hints.rs

```diff
@@ -131,15 +142,104 @@ pub fn restore_bootloader_output(
     Ok(())
 }
 
+/*
+Implements hint:
+%{
+    output_start = ids.output_ptr
+%}
+*/
+pub fn save_output_pointer_hint(
```

Comment:
For consistency: I didn't add the `_hint` suffix to the hints I implemented. Do we add it or do we remove it?

Reply:
Fixed in 2a7d5da